src/main/groovy/SDKTester.groovy

```diff
@@ -311,13 +353,32 @@ class SDKTester {
 
             // -------- commands: loadbalancer ---------
 
-            n = associate(lb, n)
-            println "associated $n with ${lb.read()}"
+            println "associated ${associate(lb, n)} with ${lb.read()}"
+            def loadBalancer = lb.read()
+
+            associatedNics(loadBalancer).collect { it }.each {
```

Comment:
this makes no sense, as you changed the call to return the entity instead - please check other incarnations of this;
on a sidenode: did you even test this?